whatToDad_site/whatToDad_app/templates/activity.html

```diff
@@ -26,9 +26,52 @@ <h1> Activities </h1>
     <br>
     <form method="POST" action="{% url 'activity' %}">
         {% csrf_token %}
-        {{ form }}
-        <input type="button" class="btn btn-secondary" value="Cancel" data-dismiss="modal"></input>
-        <input type="submit" class="btn btn-primary" name="add" value="Add activity"></input>
+        <div class="row">
+            <div class="col">
+                <h3 class="form-control">Detail: {{ form.detail }}</h3>
```

Comment:
It feels wrong to me for every field in the form to be an h3. I can see each chunk, like details, restrictions, address, etc having a header but individual fields should just be a regular element.